File: Javascript/1066.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var cpares = 0;
var cimpares = 0;
var cpositivos = 0;
var cnegativos = 0;
var n;

for (i=0;i<5;i++){
    n = parseInt(lines.shift());

    if (n%2==0) {
        cpares+=1;
    } 
    if (n%2!=0) {
        cimpares+=1;
    }
    if (n>0) {
        cpositivos+=1;
    } 
    if (n<0) {
        cnegativos+=1;
    }
}
console.log(cpares + " valor(es) par(es)");
console.log(cimpares + " valor(es) impar(es)");
console.log(cpositivos + " valor(es) positivo(s)");
console.log(cnegativos + " valor(es) negativo(s)");

File: Java/1066.java
import java.io.IOException;
import java.util.Scanner; 

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        int cpos = 0;
        int cneg = 0;
        int cimp = 0;
        int cpar = 0;
        int numero;

        for(int i=0;i<5;i++){
            numero = entrada.nextInt();
            if(numero %2==0){
                cpar++;
            } else {
                cimp++;
            }
            if(numero >0){
                cpos++;
            }
            if(numero < 0){
                cneg++;
            }
        }
        System.out.println(cpar + " valor(es) " + "par(es)");
        System.out.println(cimp + " valor(es) " + "impar(es)");
        System.out.println(cpos + " valor(es) " + "positivo(s)");
        System.out.println(cneg + " valor(es) " + "negativo(s)");
    }
}